File: 2107-find-unique-binary-string/find-unique-binary-string.py
class Solution:
    def findDifferentBinaryString(self, nums: List[str]) -> str:
        length=len(nums[0])
        y=sorted([int(i,2) for i in nums])
        res=[]
        count=0
        while True:
            if count in y:
                count+=1
            else:
                x=bin(count)[2:]
                if len(x)<length:
                    res.append('0'*(length-len(x))+x)
                    break
                else:
                    res.append(x)
                    break
        return res[-1]

File: 2107-find-unique-binary-string/find-unique-binary-string.java
class Solution {
    public String findDifferentBinaryString(String[] nums) {
       StringBuilder res = new StringBuilder();
       int n = nums.length;
       for(int i = 0 ; i < n ; i++) {
            res.append(nums[i].charAt(i) == '0' ? '1' : '0');
       }
       return res.toString();
    }
}